### ISSUE TITLE
Night CI 2022-11-17: Failed

### ISSUE THREAD

**Issue by weizhoublue** [spidernet-io/spiderpool#1060]
action url: https://github.com/spidernet-io/spiderpool/actions/runs/3491704732

Upload the zip file generated from that test failure.
[dualstack-debuglog.txt (3).zip](https://github.com/spidernet-io/spiderpool/files/10036979/dualstack-debuglog.txt.3.zip)

Copy-paste the link of the CI build where that test failure has happen.
https://github.com/spidernet-io/spiderpool/actions/runs/3491704732/jobs/5844743240

**Comment by iiiceoo** [comment]
First of all, this should be a **bug** in your code.

The CI detailed log shows that the spec failed and exited at `20:40:40.413`.

```tex
In [It] at: /home/runner/work/spiderpool/spiderpool/test/e2e/subnet/subnet_test.go:330 @ 11/17/22 20:40:40.413
```

But I observed that you created RS in your use case, but the Pod controlled by it entered cmdAdd for the first time at `20:40:40.189`.

```json
{"level":"INFO","ts":"2022-11-17T20:40:40.189Z","logger":"IPAM","caller":"ipam/ipam.go:95","msg":"Start to allocate","CNICommand":"ADD","ContainerID":"26f3719b90f2d7be042487000439602e4cca3c1625d10ce2c77164913871721a","IfName":"eth0","NetNamespace":"/var/run/netns/cni-908691c9-60e2-0931-7f7a-e07a2d7e0e6f","PodNamespace":"ns4038-142164774","PodName":"rs-4039-205812012-g5hz2"}
```

This [method](https://github.com/spidernet-io/spiderpool/blob/7dd3a17/test/e2e/subnet/subnet_test.go#L304-L310) of judging that all pods of all controllers are running is not accurate, so that the assertion was passed by mistake.

Please adjust this section and its related codes.

**Comment by ty-dc** [comment]
fix #1061
